modules/web/src/app/dynamic/enterprise/quotas/quota-widget/style.scss

```diff
@@ -81,6 +81,13 @@
         margin-bottom: 10px;
       }
     }
+
+    &.no-card {
+      left: auto;
+      right: 294px;
```

Comment:
i think the position need adjustment 
```suggestion
      right: 285px;
```

Reply:
I need to figure out why it is not looking centered on your display @ahmadhamzh, because on my laptop and monitor it is looking centered

#### Laptop (16 inches and approx 1456px width)

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/56511988/203269773-37cc3488-9a9c-4818-8dda-755d7fb7b0ea.png">

#### Monitor (32 inches and approx 2520px width)

<img width="1125" alt="image" src="https://user-images.githubusercontent.com/56511988/203269968-261d6248-a87c-4e19-bc87-cdb4bde064a9.png">

#### Reduced width via dev-tools (1040px width)

<img width="1046" alt="image" src="https://user-images.githubusercontent.com/56511988/203270286-5cd8b57a-9d4c-48dc-b369-6a6a4f46d7fc.png">